src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -1826,10 +1828,30 @@ applyTransaction (OpenView& view,
 
     try
     {
+        // If the txn signature has already been checked,
+        // then we'll get a result from the cache. If not,
+        // it'll check now (this can only happen in the case
+        // where we first see the txn during consensus,
+        // OR the cache expires). Exception: If the txn
+        // has a zero sfAccount, skip the check.
+        // The only transactions that do not need sfAccount,
+        // are also the only transactions that don't get
+        // signed. Any others will be rejected as malformed
+        // in precheck.
+        if ((*txn)[sfAccount] != zero &&
+            getApp().getHashRouter().checkValidity(*txn,
+#if RIPPLE_ENABLE_MULTI_SIGN
+            true
+#else
+            flags & tapENABLE_TESTING
+#endif
```

Comment:
As we discussed, these calls care because of testing. However, I did a quick audit of this particular call, and none of the callers can possibly have this flag set, so I'll change this to an assertion and use the default value.